RELEASE_NOTES.md

```diff
@@ -7,8 +7,78 @@
 [comment]: # (# SPDX-License-Identifier: BSD-3-Clause)
 [comment]: # (#################################################################)
 
-RAJA v0.9.0 Release Notes
-=========================
+Version v0.10.0 -- Release date 2019-xx-xx
+==========================================
+
+This release contains new features, several notable changes, and some bug fixes.
+
+Notable changes include:
+
+  * New features:
+      * Added CUDA block direct execution policies, which can be used to map
+        loop iterations directly to CUDA thread block. These are analogous to
+        the pre-existing thread direct policies. The new block direct policies
+        can provide better performance for kernels that the block loop policies
```

Comment:
Did we mean "better performance **than** the block loop policies"?

Reply:
Fixed.